databroker/core.py

```diff
@@ -384,14 +443,15 @@ def get_table(mds, fs, headers, fields=None, stream_name='primary', fill=False,
         return pd.DataFrame()
 
 
-def restream(mds, fs, headers, fields=None, fill=False):
+def restream(mds, fs, es, headers, fields=None, fill=False):
```

Comment:
The name of this function/method is now even more confusing/misleading than before. It *might* do more long-term good than harm to rethink and rename.